pypiserver/values.yaml

```
@@ -70,6 +70,13 @@ persistence:
   ## than one replica
   accessMode: ReadWriteOnce
   size: 5Gi
+  # mountPropagation: HostToContainer
+
+securityContext:
+  enabled: false
```

to keep it backward compatible it need to be enabled by default
```suggestion
  enabled: true
```